.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: "\U0001F41BBug report"
+about: "If something isn't working as expected \U0001F914."
+
+---
+
+| Question    | Answer
+| ------------| ---------------
+| Java Version | x.y.z
+| Operating System | e.g. Ubuntu/Windows/MacOS
+| Related issue | https://github.com/runelite/runelite/issues/XXX
```

Comment:
Generally if there is a related issue then a new issue should not be created.

Reply:
I can remove it. It was added because it is also mentioned within the `CONTRIBUTING.md`